.github/workflows/benchmark-runner.yml

```diff
@@ -151,4 +151,18 @@ jobs:
       benchmark_runner_group_member_id: ${{ matrix.member_id }}
       benchmark_runner_group_total: ${{ strategy.job-total }}
 
-
+  benchmark-validation:
+    needs:
+      - benchmark-search-oss-standalone
+      - benchmark-vecsim-oss-standalone
+      - benchmark-search-oss-cluster-02-primaries
+      - benchmark-search-oss-cluster-04-primaries
+      - benchmark-search-oss-cluster-08-primaries
+      - benchmark-search-oss-cluster-16-primaries
+      - benchmark-search-oss-standalone-profiler
+      - benchmark-vecsim-oss-standalone-profiler
+    runs-on: ${{ vars.RUNS_ON }}
+    if: ${{ !cancelled() }}
+    steps:
+      - if: contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')
+        run: exit 1
```

Comment:
🙏 🔥 😌

Reply:
Can this logic be added to the `event-merge-to-queue` flow under the `pr-validation` step? Possibly removing the need to specify all the benchmark flows, and won't need another step.

If not, do we still need some addition to the `pr-validation` step?

Reply:
The benchmark runner is called from other events as well (manually triggered, merge to integ branch), and I wanted to make sure it will be marked as failed.
What addition to `pr-validation` do you mean?

Reply:
The `pr-validatoin` step in the `event-merge-to-queue` basically holds all the dependencies for merging so thought it would be nice to put it there instead. That may have also removed the need to specify all the specific benchmarks, as currently general flows are depicted there.
It may not be relevant with the new changes. What do you think about not allowing merge if the benchmarks are activated but failed? I think it's good to enforce

Reply:
So I removed this step as it was unnecessary (see @GuyAv46 comment on removing `continue-on-error` and `fail-fast`). I also don't think we should fail merge if benchmarks are activated and failed... Sometimes we might be willing to compromise on it (for the sake of bug fix for instance) - the visibility is what is important.